src/alcotest.ml

```diff
@@ -93,6 +93,11 @@ module Make (M : MONAD) = struct
 
   type path = Path of (string * int)
 
+  let compare_path (Path (s, i)) (Path (s', i')) =
+    match String.compare s s' with
+    | 0 -> if i < i' then -1 else if i = i' then 0 else 1
```

Comment:
`compare i i'` would work too. Or define `let int_compare = compare` at the beginning of the file and use it and `compare` is shadowed.